src/frontend/get-github-setup.ts

```diff
@@ -5,14 +5,24 @@ import { Installation } from "../models";
 /*
 	Handles redirects for both the installation flow from Jira and
 	the installation flow from GH.
-	- From Jira: user has already installed the app and is redirected to the connect an org pg
+	- From Jira: user has already installed but wants to connect to an org not listed. Ater selecting an org in GH and giving repo permissions:
+		- they use the jiraHost we have in the cookie, connect, and are redirected to the GH config page
+		- they choose to enter an alternate jiraHost and are redirected to marketplace
 	- From GH:
-			- If we have the users Jira host, redirect to marketplace.
-			- Otherwise, render the setup page.
+		- If we have the users Jira host, redirect to marketplace.
+		- Otherwise, render the setup page (POST).
 */
 export default async (req: Request, res: Response): Promise<void> => {
 	req.log.info("Received get github setup page request");
-	const { jiraHost } = res.locals;
+	const { jiraHost, github, client } = res.locals;
+	const { data: { installations } } = await github.apps.listInstallationsForAuthenticatedUser();
+	const { data: info } = await client.apps.getAuthenticated();
+	const installationId = req.query.installation_id;
+	const installation = installations.find((item) => item.id === Number(installationId));
```

Comment:
/nit I'd probably add the casting to a number when retrieving value from the query instead of in the find: `const installationId = Number(req.query.installation_id);`